Index: app/src/main/res/layout/fragment_user_list.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/fragment_user_list.xml	(date 1602435886536)
+++ app/src/main/res/layout/fragment_user_list.xml	(date 1602435886536)
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout
+<androidx.constraintlayout.widget.ConstraintLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
@@ -44,4 +44,4 @@
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"/>
 
-</RelativeLayout>
\ No newline at end of file
+</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
Index: app/src/main/java/com/ts/tawkexam/ui/viewmodel/UserListRemoteMediator.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/ts/tawkexam/ui/viewmodel/UserListRemoteMediator.kt	(date 1602437323961)
+++ app/src/main/java/com/ts/tawkexam/ui/viewmodel/UserListRemoteMediator.kt	(date 1602437323961)
@@ -41,7 +41,9 @@
                 } else {
                     userListApi.queryUsers(
                         since = id
-                    )
+                    ).flattenAsObservable { it }
+                        .flatMapSingle { getUserObservable(it) }
+                        .toList()
                         .map { insertToDb(loadType, it) }
                         .map<MediatorResult> {
                             MediatorResult.Success(endOfPaginationReached = false)
